infra/scoped/api-documentation.tf

```diff
@@ -1102,6 +1200,20 @@ resource "aws_api_gateway_documentation_part" "users_userid_lock_put_200" {
   }
 }
 
+# 401 Unauthorized
+
+resource "aws_api_gateway_documentation_part" "users_userid_lock_put_401" {
+  rest_api_id = aws_api_gateway_rest_api.identity.id
+  properties  = file("${path.module}/api-documentation/users/:user_id/lock/put-401.json")
```

Comment:
Worth having these `properties` as a list in  a `local` and looping through em?

e.g.
```tf
locals {
  put401paths = [ "https://github.com/wellcomecollection/catalogue-pipeline/pull/1639",... ]
}
```

Reply:
Yeah, I think there is a larger (in fact, really quite large) piece of work to reduce the boilerplate in this terraform - it's all very flat at the moment which is fine but I think we could write some modules which work per-route, rather than per-terraform-resource.